Xwt/Xwt.Backends/IComboBoxCellViewFrontend.cs

```diff
@@ -27,9 +27,22 @@
 
 namespace Xwt.Backends
 {
+	public class EditableCellViewArgs : EventArgs
```

Comment:
This should be `Xwt.CellEditingFinishedArgs` (and separate file in Xwt folder and namespace) and probably even `Xwt.CellEditingFinishedArgs<T>` where T is the most convenient type (for CheckBox it would be `CheckBoxState` and not `bool` for example).

Reply:
Fixed!